### ISSUE TITLE
7 Row Keyboard

### ISSUE THREAD

**Issue by mtkunkel** [system76/laptop-suggestions#35]
### Why/User Benefit/User Problem
Ability to use function keys and hardware control keys without having to use a Fn toggle.   Fn often gets in the way of a proper ctrl key.

### Description of the feature
Dedicated row of function keys atop a backlit, robust keyboard with satisfying travel.  A taller screen ratio should help with a body size that allows for this.

**Comment by ghost** [comment]
Agreed! As a software engineer, I really hate having to use the Fn key.

**Comment by briggs1978** [comment]
I'll get behind this.  The Fn button can go straight to hell.
